### PR TITLE
Leaderboard, Result page and Question Card cpmponent

### PR DESCRIPTION
**Pull request by humzahasan**
PR Description :

Leaderboard and Result static page and Question card component added!

### CODE REVIEW

File: src/components/QuizCard/QuizCard.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        </p>
+      </div>
+      <div className="card-text">
+        <p className="card-subtitle">{question}</p>
+      </div>
+      <div className="card-action-column">
+        {allOptions.map((option, index) => (
+          <button
+            disabled={questionNum === 10}
+            key={option + Math.random()}
```
Comment (itsmebipul00) [inline src/components/QuizCard/QuizCard.jsx:29]: Why math.random here? 
Suggestion: disabled could be not hardcoded

**Review by itsmebipul00 — APPROVED** [review]
Good to merge

File: src/components/QuizCard/QuizCard.jsx (hunk 1 above)
Comment (humzahasan) [reply src/components/QuizCard/QuizCard.jsx:29]: Yup math.random can be avoided, good catch! i just added like that, will remove in final PR.

**Review by dev-prabhat — APPROVED** [review]
Looks good to me 
Good to merge

**Pull request merged**